### PR TITLE
interoperability with other clients examples

### PR DESCRIPTION
**Pull request by emasab**

### CODE REVIEW

File: examples/AvroSpecific/User.avsc [hunk 1]
```diff
@@ -1,15 +1,10 @@
-﻿{"namespace": "Confluent.Kafka.Examples.AvroSpecific",
- "type": "record",
- "name": "User",
- "fields": [
-     {"name": "name", "type": "string"},
-     {"name": "favorite_number",  "type": ["int", "null"]},
-     {"name": "favorite_color", "type": ["string", "null"]},
-     {"name": "hourly_rate", "type": {
```
Comment (mhowlett) [inline examples/AvroSpecific/User.avsc:8]: it's kind of a shame to get rid of the decimal example.
i guess it's not a big deal though, interoperability with the other examples is probably more important.

**Review by mhowlett — APPROVED** [review]
LGTM - one minor comment, then feel free to merge.

File: examples/AvroSpecific/Program.cs [hunk 2]
```diff
@@ -84,7 +83,8 @@ static void Main(string[] args)
                             try
                             {
                                 var consumeResult = consumer.Consume(cts.Token);
-                                Console.WriteLine($"user name: {consumeResult.Message.Key}, favorite color: {consumeResult.Message.Value.favorite_color}, hourly_rate: {consumeResult.Message.Value.hourly_rate}");
+                                var user = consumeResult.Message.Value;
+                                Console.WriteLine($"key: {consumeResult.Message.Key} user name: {user.name}, favorite number: {user.favorite_number}, favorite color: {user.favorite_color}");
```
[the remark this answers is not in the document]
Comment (emasab) [reply examples/AvroSpecific/Program.cs]: done

File: examples/AvroSpecific/User.avsc (hunk 1 above)
Comment (emasab) [reply examples/AvroSpecific/User.avsc:8]: I've kept it, now it's an optional field with a default value so it's compatible with the others

**Pull request merged**